bika/lims/profiles/default/metadata.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <metadata>
-  <version>1.1.0</version>
+  <version>1.1.2</version>
   <dependencies>
```

Comment:
Why the jump directly to 1.1.2 and not 1.1.1?

Reply:
I missed to set the metadata's version to 1.1.1 in my previous upgradestep :(